setup.cfg

```diff
@@ -1,4 +1,8 @@
 [tool:pytest]
+# Note: pytest settings are read here, but also in pytest.ini files.
```

Comment:
Maybe something a little different here?:
```suggestion
# Note: pytest settings may be completely overridden by a local pytest.ini file.
```

Reply:
I've added more to the comment.

Reply:
This could have a similar change as my other suggestion, dropping the first sentence, which I find misleading in the same way (without reading on).